app/models/ext_management_system.rb

```diff
@@ -433,7 +433,7 @@ def self.ems_physical_infra_discovery_types
 
   def disable!
     _log.info("Disabling EMS [#{name}] id [#{id}].")
-    update!(:enabled => false)
+    update_attribute("enabled", false)
```

Comment:
@zeari @moolitayer I had to change this since without it, when an endpoint is destroyed, the callback to remove the virtualization provider is triggered.  At that point of time there isn't an endpoint for the provider and the validations invoked by `update_attribute("enabled", false)` i.e. `hostname_format_valid?` fails since there is no endpoint to fetch the hostname from.

Reply:
I dont think we need this. It doesnt always look like it in the code but the contianers provider is valid if it has a default endpoint. So when actually using miq you should never end up  with that hostname issue.

Reply:
@zeari thanks for the hint - this change is not required.